README.md

```diff
@@ -43,8 +46,10 @@ flask run
 
 **Run flask windows**
 
-set FLASK_APP=flaskr
+cd backend
+
+$env:FLASK_APP= "flaskr"
 
-set FLASK_ENV=development
+$env:FLASK_ENV= "development"
 
 flask run
```

Comment:
`set FLASK_APP=flaskr` will work on the Windows normal **command prompt**.

The `$env:FLASK_APP="flaskr"` is for the **Windows Power Shell**. Let's add both of them in.